examples/hello.rs

```diff
@@ -37,29 +37,30 @@ fn main() -> std::io::Result<()> {
 
     // We also create the annotations list here that allows us to have things
     // like links or comments on the page.
-    let mut annots = page.annotations();
+    let mut annotations = page.annotations();
 
     // Write a new annotation.
-    let mut annot = annots.add();
+    let mut annotation = annotations.push();
 
     // Write the type, area, and action for this annotation.
```

Comment:
I feel like this comments lists a random portion of the things that are actually configured.